great_expectations/data_context/cloud_migrator.py

```diff
@@ -101,6 +119,11 @@ class ConfigurationBundleSchema(Schema):
     _data_context_variables = fields.Nested(
         DataContextConfigSchema, allow_none=False, data_key="data_context_variables"
     )
+    _datasources = fields.List(
+        fields.Nested(DatasourceConfigSchema, allow_none=True, required=True),
+        required=True,
+        data_key="datasources",
+    )
```

Comment:
Do we need the prefix `_` for schema fields? I think the access modifier aspect of the bundle can be determined with the runtime object no?

Reply:
Not necessary if we add accessors, sounds like you are doing this already for the printing logic so we can swap these out as well.

Reply:
Added! (and modified the schema)